scripts/ayp.coffee

```diff
@@ -312,13 +312,17 @@ class AYPStrip
 
     @loadAvatars names, (err, avatars) =>
       return cb(err, null) if err
+      scrambleLines = []
 
       if names.length == 1
         # The only person speaking is centered in the frame
         char = avatars[names[0]]
         left = (frame.width / 2) - (char.width / 2)
         top = (frame.height - char.height)
         @compositeImage frame, char, left, top
+
+        isKenny = char.matchchar.match('default') == null
```

Comment:
I didn't look too closely at how the avatar comes back and thought it'd come back as a path. Would your proposal work?